src/main/beginner_mode/beginner_mode.c

```diff
@@ -0,0 +1,29 @@
+#include "platform.h"
```

Comment:
Please add the standard header (as in https://github.com/betaflight/betaflight/blob/f00e4f2a82edbadbd5517e6d8f4b0f78231d4a63/src/main/main.c#L1-L16) to all new files.
Also, this should go into `src/main/flight`, and not into a new subdirectory.